### ISSUE TITLE
Fix background / foreground color contrast

### ISSUE THREAD

**Issue by TheRealAlan** [CastroForGeorgia/GA-5#50]
According to the Lighthouse audit, the background / foreground contrast in the hero of the homepage doesn't have sufficient contrast. This is an accessibility issue.
